universal-application-tool-0.0.1/conf/evolutions/default/12.sql

```diff
@@ -0,0 +1,9 @@
+# --- Add versioning and lifecycle management.
+
+# --- !Ups
+
+alter table questions add repeater_id jsonb;
```

Comment:
Question (blocking): question IDs are integers, why use `jsonb` here instead of `bigserial`?

Reply:
Switched to storing a nullable bigserial that is converted to Optional<Long> in QuestionDefinition.